Orso.Arpa.Domain/UserDomain/Commands/RefreshAccessToken.cs

```diff
@@ -83,7 +92,22 @@ public async Task<string> Handle(Command request, CancellationToken cancellation
                         "than the one with which the token was created. The refresh token has been revoked. Please log in again.");
                 }
 
-                return await _jwtGenerator.CreateTokensAsync(user, request.RemoteIpAddress, cancellationToken);
+                var token = await _jwtGenerator.CreateTokensAsync(user, request.RemoteIpAddress, cancellationToken);
+
+                List<Claim> claims = new List<Claim>{
+                    new Claim("JwtToken", token)
+                };
+
+                var claimsIdentity = new ClaimsIdentity(
+                    claims,
+                    CookieAuthenticationDefaults.AuthenticationScheme);
+
+                var signInTask = _httpContextAccessor.HttpContext.SignInAsync(
```

Comment:
` _signInManager.RefreshSignInAsync(user)` is not updating the cookie claims so this is the option if claims should be updated also